README.md

```diff
@@ -45,7 +45,7 @@ OpenDevin agents collaborate with human developers to write code, fix bugs, and
 ![App screenshot](./docs/static/img/screenshot.png)
 
 ## ⚡ Getting Started
-The easiest way to run OpenDevin is inside a Docker container. It works best with the most recent version of Docker, `26.0.0`.
+OpenDevin works best with the most recent version of Docker, `26.0.0`.
```

Comment:
The text made sense, actually. The installed Docker pacakage in Linux (WSL/Ubuntu etc) is currently 27.0.3.
But for Windows users, there's also Docker Desktop that has version 4.32.
Could we work that distinction in maybe?

Reply:
Let me try.

The reason I made this change is because in slack someone mentioned that saying "The easiest way to run OpenDevin is inside a Docker container" implicitly means that there is a way you can run OpenDevin without a docker container. Which isn't quite true. You need docker for the sandbox. What do you think?

Reply:
Heh I've read that sentence dozens of times, but now that you mention it, a proper clarification makes perfect sense, indeed! Good catch!

Reply:
@tobitege I'm going to merge this PR cause it's release related and then have a follow up PR regarding this README.

Looking at my own Docker Desktop, I'm now running Docker Engine 27.0.0 and this README says 26.0.0. So would saying something like:
"OpenDevin works best with Docker engine version 26.0.0+ (Docker Desktop 4.31.0+)" make sense?

Reply:
Yes, sounds good!